OsuPlayer/Views/PlaylistViewModel.cs

```diff
@@ -80,7 +80,10 @@ public PlaylistViewModel(IPlayer player)
         {
             if (d.NewValue != RepeatMode.Playlist)
             {
-                _materialIcons.ForEach(x => x.IsVisible = false);
+                Dispatcher.UIThread.Post(() =>
+                {
+                    foreach (var x in _materialIcons) x.IsVisible = false;
```

Comment:
I think a line break isn't needed here for sure, but I think one here would look better. 

Up to you